tests/by-util/test_fmt.rs

```diff
@@ -33,7 +33,19 @@ fn test_fmt_width() {
         new_ucmd!()
             .args(&["one-word-per-line.txt", param, "10"])
             .succeeds()
-            .stdout_is("this is\na file\nwith one\nword per\nline\n");
+            .stdout_is("this is a\nfile with\none word\nper line\n");
+    }
+}
+
+#[test]
+fn test_small_width() {
+    for width in ["1", "2", "3"] {
```

Comment:
There is one value missing that unfortunately makes the test fail: `0`.

Reply:
Interesting. I don't see this on the man or info pages, but I'm guessing GNU `fmt` treats `-w0` as `-w1`?

Running this in the uutils/coreutils repo gave no output:
```
fd -t f -0 | while IFS= read -r -d '' file; do diff <(fmt -0 "$file") <(fmt -1 "$file"); done
```

I'm not sure whether to special case treating a zero width as one width, or treating the goal as zero in that case (both fix this test when run with `"0"`). I'm pushing a PR that changes the calculated goal since that seems like the smaller change, but I'm not sure either are different